examples/typescript/package.json

```diff
@@ -0,0 +1,213 @@
+{
+  "dependencies": {
+    "react": "^0.14.3",
+    "react-dom": "^0.14.3"
+  },
+  "devDependencies": {
+    "jest-cli": "^0.8.0",
+    "react-addons-test-utils": "^0.14.3",
+    "typescript": "^1.6.2"
+  },
+  "scripts": {
+    "test": "jest"
+  },
+  "jest": {
+    "preprocessCachingDisabled": true,
```

Comment:
o module version
Except for typescript version (<1.6), I think there is no restrictions.

o Jest preprocessCachingDisabled
Should be deleted.

I will fix and test soon.